### ISSUE TITLE
Issues going to town and then back through portal

### ISSUE THREAD

**Issue by bhhandley** [Kryszard-POD/Kryszard-s-PD2-Loot-Filter#15]
I'm guessing this has to do with the new in-town feature but I've suddenly been having an issue with all items showing when I go back through a portal from town.  The only way I've found to fix it is to open the settings and change the filter level back and forth.

The most common way I've reproduced this is doing trav runs.  Portal to town after killing, drop gold at chest, go back into portal.

**Comment by Kryszard-POD** [comment]
Hey. 
Yes I am aware of this issue - and yes, it is related to the newest update with showing all items in towns.
It is working in a weird way - those extra visible items gonna dissappear once you get away from the spot (by walking away) and when you'll go back to this location items gonna be hidden again. The issue appears only when you go back to the fight field with town portal.
If this gonna be annoying for many players I am gonna remove this feature I guess. We will see.

**Comment by bhhandley** [comment]
Yep, makes sense.  Not a huge deal, just wanted to report it -- not sure why I didn't think about it before but I have my reload config bound, that fixes it too.

Thanks for the great filter.
